src/scripts/discord.ts

```diff
@@ -60,7 +60,12 @@ const getActivity = (): Presence => {
       presence.state = mediaInfo.artists ? mediaInfo.artists : "unknown artist(s)";
       presence.largeImageKey = mediaInfo.image;
       if (mediaInfo.album) {
-        presence.largeImageText = mediaInfo.album;
+        if (mediaInfo.album.length < 2) {  // Fix for DiscordRPC 2 character minimum
+          presence.largeImageText = mediaInfo.album + " ";
+        }
+        else {
+          presence.largeImageText = mediaInfo.album;
+        }
```

Comment:
This could have been a tenary:

```js
presence.largeImageText = mediaInfo.album.length < 2 ? mediaInfo.album + " " : mediaInfo.album;
```

Reply:
Yeah that works too 👍
I'm not the best at typescript, I just wrote what made sense to me for something to fix my issue and worked, but any changes are welcome if it looks neater etc...